indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,324 @@
+{% macro render_program(event, days, config, program_config) %}
+    {% for day, entries in days.items() %}
+        <div class="day pagebreak program-{{loop.index}} {{ 'pagebreak-children' if config.new_page_per_session else '' }}" id="day-{{ loop.index }}">
+            <div class="day-title">
+                <h2>{{ day|format_skeleton('EEEEdMMMM', timezone=event.tzinfo) }}</h2>
+            </div>
+            {% for entry in entries %}
+                {% set show_children_location = program_config.show_children_location[entry.id] %}
+                {% set should_render = (
+                    entry.type.name == 'SESSION_BLOCK' or
+                    (entry.type.name == 'CONTRIBUTION' and config.show_contribs) or
+                    (entry.type.name == 'BREAK' and config.show_breaks)
+                ) %}
+
+                {% if should_render %}
+                    <div class="day-entry">
+                        {% if entry.type.name != 'CONTRIBUTION' %}
+                            {{ _get_color_styles(entry) }}
+                        {% endif %}
+                        <div class="time-text time-from">
+                            {{ entry.start_dt|format_time(timezone=event.tzinfo) }}
+                        </div>
+                        <section class="{{ _get_custom_id(entry) }}">
+                            {{ _render_block_date(entry, day, config.print_date_close_to_sessions) }}
+                            <section class="session-block {{ 'contribution' if entry.type.name == 'CONTRIBUTION' else '' }}">
+                                <div class="session-block-wrapper">
+                                    {% if entry.type.name == 'BREAK' %}
+                                        {{ _render_break(entry, config, program_config) }}
+                                    {% elif entry.type.name == 'CONTRIBUTION' %}
+                                        {{
+                                            _render_contrib(
+                                                entry,
+                                                config,
+                                                program_config,
+                                                show_children_location
+                                            )
+                                        }}
+                                    {% else %}
+                                        {{
+                                            _render_session_block(
+                                                entry,
+                                                config,
+                                                program_config,
+                                                show_children_location
+                                            )
+                                        }}
```

Comment:
I think it's short enough to look better?

```suggestion
                                        {{ _render_session_block(entry, config, program_config, show_children_location) }}
```

Anyway, not a super strong opinion here so if you prefer the wrapped version feel free to keep that one (and likewise a few lines above).